plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeGlueMetastore.java

```diff
@@ -241,6 +252,37 @@ public void testHideNonDeltaLakeTable()
                 .isEmpty();
     }
 
+    @Test
+    public void testDoubleCreateSchemaForTheSameSession()
+    {
+        String testSchemaName = "test_schema_" + randomName();
+
+        DeltaLakeMetadata metadata = metadataFactory.create(session.getIdentity());
+
+        metadata.createSchema(session, testSchemaName, Map.of(), TRINO_PRINCIPAL);
+        metadata.createSchema(session, testSchemaName, Map.of(), TRINO_PRINCIPAL);
```

Comment:
Maybe call `metadata.schemaExists()` after the second call to be clear that the schema exists before doing the second call.